target/appengine-staging/__static__/business.js

```diff
@@ -0,0 +1,17 @@
+//implementing map
```

Comment:
Same with this file. THis might overwrite sara's changes. What is appengine-staging?